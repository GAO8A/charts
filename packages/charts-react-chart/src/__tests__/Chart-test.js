import React from 'react';
import renderer from 'react-test-renderer';
import Chart from '../';

describe('Chart Component', () => {
  it('should render', () => {
    const tree = renderer.create(
      <Chart />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
