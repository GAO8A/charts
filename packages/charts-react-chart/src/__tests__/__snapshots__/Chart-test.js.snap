exports[`Chart Component should render 1`] = `
<svg
  className={undefined}
  height={500}
  width={960}>
  <g
    transform="translate(50,20)" />
</svg>
`;
